packages/marshal/src/deeplyFulfilled.js

```diff
@@ -11,6 +11,7 @@ import { X, q } from '@endo/errors';
 const { ownKeys } = Reflect;
 const { fromEntries } = Object;
 
+// TODO return a type conditioned on the parameter as deeplyFullfilledObject from agoric-sdk does
```

Comment:
Optional wordsmithery.
```suggestion
// TODO return a type contingent on the parameter as deeplyFullfilledObject from agoric-sdk does
```